src/code/z_sub_s.c

```diff
@@ -781,8 +1194,8 @@ s32 SubS_CopyPointFromPathList(Path* paths, s32 pathIndex, s32 pointIndex, Vec3f
     return false;
 }
 
-u8 SubS_GetPathCount(Path* paths, s32 index) {
-    Path* path = &paths[index];
+u8 SubS_GetPathCountFromPathList(Path* paths, s32 pathIndex) {
+    Path* path = &paths[pathIndex];
 
     return path->count;
```

Comment:
I also think in the `Path` struct, `count` -> `numPoints` since `count` makes me think of an incrementer that can change, while `num` makes me think more of "total"